web/src/components/AnalyzeData.vue

```diff
@@ -0,0 +1,35 @@
+<script>
+import Vue from 'vue';
+import { analyses } from './vis';
+
+export default Vue.extend({
```

Comment:
Why this `Vue.extend()` idiom here, instead of just exporting the object?

Reply:
fixed